shared/teams/team/tabs/index.tsx

```diff
@@ -63,7 +63,11 @@ const TeamTabs = (props: TeamTabsProps) => {
   }
 
   if (props.numSubteams > 0 || props.showSubteams) {
-    tabs.push(<TabText key="subteams" selected={props.selectedTab === 'subteams'} text="Subteams" />)
+    tabs.push(
```

Comment:
why wasn't this wrapped in a Kb.Box before?

Reply:
Yeah it's odd; doesn't seem like it was intentional